src/explorer/model/OmittedStatus.ts

```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT license.
 
 export interface IOmittedStatus {
```

Comment:
I will refine `omittedStatus?` soon.

Reply:
Before you start to do refactoring, create an issue, describe why and how you are going to do.